tests/gtest_like_c.c

```diff
@@ -0,0 +1,31 @@
+#include "gtest_like_c.h"
+
+jmp_buf gtest_jmp;
+
+int gtest_like_c_run_tests(const struct Test* tests, size_t num_tests) {
+    int passed_tests = 0;
+    int skipped_tests = 0;
+
+    for (int i = 0; i < num_tests; i++) {
+        switch (setjmp(gtest_jmp))
+        {
+        case TEST_PASSED:
+            printf("Test %s %s\n", tests[i].group_name, tests[i].test_name);
+            tests[i].fn_test();
+            printf("passed\n");
```

Comment:
Assuming using already existing C-only testing framework is not an option and assuming there is no implementation of "GTest for C" already, it would make sense to produce the same output as the GTest.

Reply:
Output has been improved to follow gtest output.